corehq/apps/analytics/tasks.py

```diff
@@ -393,3 +396,9 @@ def _log_response(data, response):
         logger.error(message)
     else:
         logger.debug(message)
+
+
+def _get_ab_test_properties(user):
```

Comment:
i was planning on building out this dictionary as we add new properties, the others could have slugs but it seemed like there was no reason to add it to the first one.
Longer term i would love for these to be in a postgres table so we could query them or add new ones through a ui, but that seemed overkill for the interrupt ticket, just something to add to my analytics backlog.